astropy/constants/iau2012.py

```diff
@@ -37,7 +37,7 @@ class IAU2012(Constant):
 
 # Luminosity
 L_bol0 = IAU2012('L_bol0', "Luminosity for absolute bolometric magnitude 0",
-                 3.0128e28, "W", 0.0, "IAU 2015 Resolution B 2", system='si')
+                 3.0128e28, "W", 0.0, "IAU 2012 Resolution B 2", system='si')
```

Comment:
I think this one is worse - the value is indeed from 2015, so in 2012 it must have been another one (with an error bar still)....

Reply:
Hmm. I don't see this constant mentioned in https://www.iau.org/static/resolutions/IAU2012_English.pdf

Reply:
Could it be that this thing didn't exist until 2015 and it is inserted into `iau2012` for API compatibility? Maybe @stargaser remembers. https://arxiv.org/abs/1510.06262

Reply:
Hmm, doing a bit of digging, I see that in astropy 1.0 and 1.3, we had
```
L_sun = Constant('L_sun', "Solar luminosity", 3.846e26, 'W', 0.0005e26,
                 "Allen's Astrophysical Quantities 4th Ed.", system='si')
```
So, I think that is what `iau2012` should have as well.

Reply:
But... it is a breaking change. What to do?

Reply:
@mhvk , but the line in question is `L_bol0`, not `L_sun`.

Reply:
Duh, clearly not reading very carefully. So, `L_bol0` is there in astropy 1.3, with the same docstring. That was 2016, so by then `L_bol0` was indeed known; I think it just ended up in `iau2012`, since that is mostly what was used in 1.3.

But in any case, it was fixed in 2015!

Reply:
@mhvk , so do you think the comments added are okay then? I agree that no code change is needed and not a bug (phew!) but hopefully these comments would prevent this confusion from resurfacing in the future.